components/infobox/wikis/apexlegends/infobox_weapon_custom.lua

```diff
@@ -168,6 +202,10 @@ function CustomWeapon:_createContextualNoWrappingSpan(content, index, lookUpTabl
 	return tostring(span)
 end
 
+function CustomWeapon:_createContextualNoWrappingNote(noteText)
```

Comment:
This function seems to do no `no wrapping´, why is it called that?

Reply:
Sorry, that was myself copying the other No Wrapping function over for speed and forgetting to change the name properly